domains/frenchguy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "A dumb kid who likes da computer",
+  "repo": "https://github.com/nvct-website/nvct-website.github.io",
+  "owner": {
+    "username": "Noahvocat (also known as NVCT, Noahvocado or Noahvocados or FrenchGuy or AverageFrenchGuy or AvrgFrnchGuy)",
+    "email": "",
+    "discord": "avrgfrnchguy"
+  },
+  "record": {
+    "CNAME": "nvct-website.github.io"
```

Comment:
This site has redirection to the domain which you are requesting to delete as well.

Reply:
I just removed the domain from the website. It’s just a Github Pages 404 now.

Reply:
So is there any content on the site or no?